core/src/main/java/io/opentelemetry/android/export/DelegatingExporter.kt

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.export
+
+import android.util.Log
+import io.opentelemetry.android.common.RumConstants.OTEL_RUM_LOG_TAG
+import io.opentelemetry.api.internal.GuardedBy
+import io.opentelemetry.sdk.common.CompletableResultCode
+import java.nio.BufferOverflowException
+import kotlin.math.log
```

Comment:
nit: it doesn't seem like we need this import.

Reply:
Fixed